### DISCUSSION TITLE
Plugin API: ToolEventArgs mapCoords/screenCoords blank?

### DISCUSSION THREAD

**Discussion by hawkerm** [OpenRCT2/OpenRCT2#19147] [Q&A] [answered]
Hi,

I'm trying to work on a mod and get the coordinates of the mouse with a tool in the onDown/onMove/onUp events.

However, they appear to be blank:

![image](https://user-images.githubusercontent.com/8959496/212526633-2f9c9837-7be3-4d9b-a736-321103926a72.png)

Is there something I'm missing to enable these or this a bug?

I couldn't see where this event/args got created in the codebase.

Thanks!

**Comment by hawkerm** [comment, marked as answer]
Nevermind, I was trying to output to chat message (as I didn't have the terminal open with com), would be nice of console.log was echoed to dev console in game.

Anyway, found out chat strips `{` characters, so that was causing issue with me seeing values.

**Reply by IntelOrca** [reply]
Braces are used for formatting the text, I think two braces `{{` might escape them.
